ci/scripts/java_jni_manylinux_build.sh

```diff
@@ -35,9 +35,6 @@ echo "=== Clear output directories and leftovers ==="
 rm -rf ${build_dir}
 
 echo "=== Building Arrow C++ libraries ==="
-devtoolset_version=$(rpm -qa "devtoolset-*-gcc" --queryformat %{VERSION} | \
-                       grep -o "^[0-9]*")
-devtoolset_include_cpp="/opt/rh/devtoolset-${devtoolset_version}/root/usr/include/c++/${devtoolset_version}"
```

Comment:
@jonkeane Is it expected change?

Reply:
Aaaah, I might have been overly aggressive in removing from this file. 

I thought we only used the `devtoolset_version` on the R CentOS 7 builds and that other places we added that to were borrowing from those and no longer necessary. But if we need to add it back here that's totally fine. Sorry for not confirming that and the hassle.

Reply:
No problem. :-)
Thanks for confirming this.